bin/jade.js

```diff
@@ -58,6 +59,17 @@ program.on('--help', function(){
 
 program.parse(process.argv);
 
+// locals given, parse them
+
+var locals;
+if (program.locals) {
+  if (exists(program.locals)) {
+    locals = JSON.parse(fs.readFileSync(program.locals));
```

Comment:
Didn't mean that to come across as a complaint.  It's always good to get feedback, just thought if you were reviewing the code I didn't want you to waste your time :)